docs/source/bootstrapping.md

```diff
@@ -23,7 +24,12 @@ Options:
                                   deploy  [default: no-synth]
   --profile TEXT                  The AWS profile to initiate a session
   --region TEXT                   AWS region to use
-  --qualifier TEXT                A qualifier to append to toolchain role (alpha-numeric char max length of 6)
+  --qualifier TEXT                A qualifier to append to toolchain role
+                                  (alpha-numeric char max length of 6)
+  -pa, --policy-arn TEXT          ARN of existing Policy to attach to Target
```

Comment:
can you make this more clear about it being valid in single account structure (`--as-target`)?

Reply:
I can